app/facades/tutorial_facade.rb

```diff
@@ -27,6 +27,6 @@ def current_video_index
   end
 
   def maximum_video_position
-    videos.max_by { |video| video.position }.position
+    videos.max_by(&:position).position
```

Comment:
I don't know how this code words but it seems to work...whenever I see an ampersand I'm like "cool! What does that do again?"

Reply:
I'm right there with you. But I believe it is shorthand to writing single line iteration
`videos.max_by { |video| video.position}` == `videos.max_by(&:position)`
We should make an effort to try and use it more often